demo/cavity_flow.py

```diff
@@ -30,12 +30,10 @@ def set_boundary_conditions(self):
         no_slip = VelocityBCType.no_slip
         constant = VelocityBCType.constant
         BoundaryMarkers = HyperCubeBoundaryMarkers
-        velocity_bcs = (
-                (no_slip, BoundaryMarkers.left.value, None),
-                (no_slip, BoundaryMarkers.right.value, None),
-                (no_slip, BoundaryMarkers.bottom.value, None),
-                (constant, BoundaryMarkers.top.value, (1.0, 0.0)))
-        self._bcs = {"velocity": velocity_bcs}
+        self._bcs = ((no_slip, BoundaryMarkers.left.value, None),
+                     (no_slip, BoundaryMarkers.right.value, None),
+                     (no_slip, BoundaryMarkers.bottom.value, None),
+                     (constant, BoundaryMarkers.top.value, (1.0, 0.0)))
```

Comment:
Why we have these changes about the definition of boundary conditions

Reply:
The reason for this change is due to changes in the underlying problem and solver classes. The way the boundary conditions are specified has changed in this branch.

Reply:
Moreover, it is clear from
```
(VelocityBCType.no_slip, BoundaryMarkers.left.value, None)
```
that this is a velocity boundary condition. Therefore, the mapping is not really required.